internal/catalogmetadata/filter/successors.go

```diff
@@ -60,6 +60,11 @@ func legacySuccessor(installedBundle ocv1.BundleMetadata, channels ...declcfg.Ch
 			}
 		}
 		if candidateBundleEntry.SkipRange != "" {
+			// There are differences between how "github.com/blang/semver/v4" and "github.com/Masterminds/semver/v3"
+			// handle version ranges. OLM v0 used blang and there might still be registry+v1 bundles that rely
+			// on those specific differences. Because OLM v1 fully supports registry+v1 bundles,
```

Comment:
```suggestion
			// on those specific differences. Because OLM v1 supports a sub-set of registry+v1 bundles,
```

Reply:
I think people might wonder what sub-set would that be exactly and if we already only support a sub-set, then why not the subset of bundles that expects versioning handled the way 'masterminds' and not 'blang' handles it?

Additional ideas:
1. ~"Because OLM v1 fully supports registry+v1 bundles versioning" - I think that's true and is the specific reason we are keeping 'blang' as I understood.
2. add ~"limited" and link to the limitations @camilamacedo86 linked to

Open to additional opinions of course, but I don't think we should be overthinking this here :)

Reply:
Anyway that we do not write that we fully support in OLM v1 is OK for me 👍

Reply:
I went with just removing 'fully' - the remaining statement is true and basically taken verbatim from out internal discussion

While 'sub-set' might be good to add in other contexts, it didin't make sense for me in this one - it would essentially be something like (paraphrasing) "because olm v1 supports a sub-set of bundles, we need to keep using blang", which to me would imply that if we didin't use blang we would be supporting not a sub-set but a full range of bundles.